resources/views/backend/movies/create.blade.php

```diff
@@ -0,0 +1,147 @@
+                        @enderror
+                            <input type="text" class="form__input" placeholder="{{ trans('release_year') }}(*)" name="release_year" value="{{ old('release_year') }}">
+                        </div>
+
+                        <div class="col-12 col-sm-6 col-lg-3">
+                        @error('runtime')
+                            <div class="main__table-text--red">{{ $message }}</div>
+                        @enderror
+                            <input type="text" class="form__input" placeholder="{{ trans('running_time') }} ({{ trans('minute') }})(*)" name="runtime" value="{{ old('runtime') }}">
+                        </div>
+
+                        <div class="col-12 col-sm-6 col-lg-3">
+                        @error('quality')
+                            <div class="main__table-text--red">{{ $message }}</div>
+                        @enderror
+                            <select class="js-example-basic-single" id="quality" name="quality">
+                            @foreach ($qualities as $quality)
+                                <option value="{{ $quality['order'] }}" @if( $quality['order'] == old('quality') ) selected @endif>{{ $quality['name'] }}</option>
+                            @endforeach
+                            </select>
+                        </div>
+
+                        <div class="col-12 col-sm-6 col-lg-3">
+                        @error('age')
+                            <div class="main__table-text--red">{{ $message }}</div>
+                        @enderror
+                            <input type="text" class="form__input" placeholder="{{ trans('age') }}(*)" name="age" value="{{ old('age') }}">
+                        </div>
+
+                        <div class="col-12 col-sm-6 col-lg-3">
+                        @error('country')
+                            <div class="main__table-text--red">{{ $message }}</div>
+                        @enderror
+                            <select class="js-example-basic-multiple" id="country" name="country">
+                            @foreach ($countries as $country)
+                                <option value="{{ $country['order'] }}" @if( $country['order'] == old('country') ) selected @endif>{{ trans($country['name']) }}</option>
+                            @endforeach
+                            </select>
+                        </div>
+
+                        <div class="col-12 col-sm-6 col-lg-3">
+                        @error('genre')
+                            <div class="main__table-text--red">{{ $message }}</div>
+                        @enderror
+                            <select class="js-example-basic-multiple" id="genre" name="genre">
+                            @foreach ($genres as $genre)
+                                <option value="{{ $genre['order'] }}" @if( $genre['order'] == old('genre') ) selected @endif>{{ trans($genre['name']) }}</option>
```

Comment:
```suggestion
                                <option value="{{ $genre['order'] }}" @if ( $genre['order'] == old('genre') ) selected @endif>{{ trans($genre['name']) }}</option>
```
tương tự, sửa hết các lỗi tương tự